### ISSUE TITLE
the question about your code “logits = model(hg_atk, features)”

### ISSUE THREAD

**Issue by lxw106** [MengmeiZ/RoHe#3]
Thanks for the opening source. “logits = model(hg_atk, features)”,Is the “model” in this code trained using the original graph or the perturbation graph?Looking forward to your reply very much！！!

**Comment by MengmeiZ** [comment]
We focus on the setting of evasion attack. So all the heterogeneous GNNs are trained on original graph and are tested on perturbation graph.

**Comment by lxw106** [comment]
非常感谢您的回复



---Original---
From: ***@***.***&gt;
Date: Thu, Sep 14, 2023 16:17 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [BUPT-GAMMA/RoHe] the question about your code “logits = model(hg_atk, features)” (Issue #3)




 
We focus on the setting of evasion attack. So all the heterogeneous GNNs are trained on original graph and are tested on perturbation graph.
